iac-extensions/spring-config/src/test/java/org/sonar/iac/springconfig/utils/SpringConfigVerifier.java

```diff
@@ -54,14 +54,17 @@ public static void verifyNoIssue(String filename, IacCheck check) {
   }
 
   private static InputFileContext createContextForFile(String filename) {
-    var file = IacTestUtils.inputFile(filename, BASE_DIR);
+    var language = "";
+    if (filename.endsWith(".yaml") || filename.endsWith(".yml")) {
+      language = "yaml";
+    }
+    var file = IacTestUtils.inputFile(filename, language);
```

Comment:
Apparently it isn't really important or validated somewhere, but this call uses a different path than `BASE_DIR`
I would change it to  `IacTestUtils.inputFile(filename, BASE_DIR, language)` and also make change line 57 to `var language = null;` (`String language;`) as it's nullable.